src/WCSimPMTObject.cc

```diff
@@ -3113,7 +3113,8 @@ G4double PMT3inchR14374::GetPMTGlassThickness() {return 0.1*cm;}     // TF: from
 
 // Currently based on 8" (instead of 20")
 // But shifted to requirements (2ns TTS FWHM) for 1 pe
-double PMT3inchR14374::HitTimeSmearing(double Q, double TTSFF=1.0) {
+// Implementation is not charge-dependent---no need for argument name Q.
+double PMT3inchR14374::HitTimeSmearing(double, double TTSFF=1.0) {
```

Comment:
I committed a variation of the comments for the TTSFF argument.  See [L3148-L3153](https://github.com/spradlin/WCSim/blob/6829ddaed08e34298846b1db6485608bc078eded/src/WCSimPMTObject.cc#L3148-L3153):
```
// Implementation of pure-virtual base-class method
//   double WCSimPMTObject::HitTimeSmearing(double, double)
// The first argument is usually a charge, Q, but that quantity is not used
// in this implementation.  The unused first argument is anonymous to prevent
// compiler warnings about unused arguments.
double PMT3inchR14374::HitTimeSmearing(double, double TTSFF=1.0) {
```